docs/docs/post-css.md

```diff
@@ -2,7 +2,53 @@
 title: PostCSS
 ---
 
-This is a stub. Help our community expand it.
+PostCSS transforms extended syntaxes and features into modern, browser-friendly CSS. This guide will show you how to get started with Gatsby and PostCSS.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Installation and Configuration
+
+1.  Install the Gatsby plugin **gatsby-plugin-postcss**.
```

Comment:
Convenient to link the plugin title to the plugin source code, perhaps? https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-postcss